setup.sh

```diff
@@ -126,14 +126,6 @@ if ! hash ansible-playbook &> /dev/null; then
     pip install --upgrade --editable git+https://github.com/ansible/ansible.git@devel#egg=ansible
 fi
 
-# User doesn't have a default virtualenv, let's configure one
-if [ ! -f ~/.bashrc ] || ! grep 'source.*activate' ~/.bashrc; then
-    echo '# Activate ansible virtualenv' >> ~/.bashrc
-    echo 'source ~/.ansible-env/bin/activate' >> ~/.bashrc
-    echo 'export ANSIBLE_STDOUT_CALLBACK=debug' >> ~/.bashrc
-    echo '!! You need to login again for changes to take effect !!'
-fi
```

Comment:
While removing this may be interresting for python hackers who don't read the script before executing it, is it possible to provide a replacement feature for non python hackers (ie. our beloved scala team) ? Thanks.